### ISSUE TITLE
missing dependency on default?

### ISSUE THREAD

**Issue by Lemente** [sfence/clothing#11]
I am getting this error without the default mod

```
ModError: Failed to load and run script from C:\Users\Lem\Desktop\minetest-5.6.0-win64\bin\..\mods\clothing\init.lua:
...esktop\minetest-5.6.0-win64\bin\..\mods\clothing/api.lua:3: attempt to index global 'default' (a nil value)
stack traceback:
	...esktop\minetest-5.6.0-win64\bin\..\mods\clothing/api.lua:3: in main chunk
	[C]: in function 'dofile'
	...sktop\minetest-5.6.0-win64\bin\..\mods\clothing\init.lua:3: in main chunk
Check debug.txt for details.```

**Comment by sfence** [comment]
Thanks for reporting.

With which game you are trying to use clothing mod?

**Comment by Lemente** [comment]
A custom one

**Comment by sfence** [comment]
Fixed with commit 09985c63894799ec08475eccc9ccf405b9af05ac
